src/Propagation/ServerTiming/src/ResponsePropagator.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\Propagation\ServerTiming;
+
+use OpenTelemetry\Context\ContextInterface;
+use OpenTelemetry\Context\Propagation\PropagationSetterInterface;
+
+/**
+ * This propagator type is used to inject the trace-context into HTTP responses.
+ */
+interface ResponsePropagator
```

Comment:
I think this is not required, and it should use core's `TextMapPropagatorInterface` instead